src/pages/programDisaggregations/form/CategoryMappingSection.tsx

```diff
@@ -152,11 +156,23 @@ const useCategories = ({
 
         // A suggested category is a category that is part of a selected categoryCombo
         // in a program indicator mapping, but not already mapped in the form.
+
+        // keep track of categories included to avoid duplicates
+        const alreadyIncludedInSuggestedCategories: { [key: string]: boolean } =
```

Comment:
the modifications here also fix another bug I noticed, where you could end up with duplicate suggestions for categories to add:
<img width="1034" alt="image" src="https://github.com/user-attachments/assets/09d59c3d-a5c6-4a9c-978c-73f29c2298c4" />